plugins/profiles.koplugin/main.lua

```diff
@@ -41,6 +43,25 @@ function Profiles:onFlushSettings()
     end
 end
 
+local function dispatcherRegisterProfile(name, unregister)
+    if unregister then
+        Dispatcher:removeAction("profile_exec_"..name)
+        Dispatcher:removeAction("profile_menu_"..name)
+    else
+        Dispatcher:registerAction("profile_exec_"..name,
+            {category="none", event="ProfileExecute", arg=name, title=T(_("Profile execute: %1"), name), general=true})
+        Dispatcher:registerAction("profile_menu_"..name,
+            {category="none", event="ProfileShowMenu", arg=name, title=T(_("Profile show menu: %1"), name), general=true})
```

Comment:
```
Profile execute: %1
Profile show menu: %1
```
These "sounds" a bit programmatically, and not super nice English.
No real alternative idea though :/

Also, "menu" is our internal name for such lists (for a user, Menu is something at the top, that pops down :), dunno if a better name could be used, like "popup", "popup list".
QuickMenu (below) sounds a bit better as a nickname for that thing.

Reply:
What exactly do they mean?

Reply:
```
Execute profile.
Show profile as a menu/popup.
```
(I guess the current naming is to have them both started with `Profile...` to make it obvious they come from the same Profile plugin.

Reply:
I can see the attraction, but I think it'd be better to simply name them that.

Reply:
The first one could kinda work as `Profile execution` but I don't have anything better for the second than `Profile menu`, and both are arguably worse than the verbose approach ;).

Reply:
![11](https://user-images.githubusercontent.com/62179190/190729167-ca318638-8cfd-43e8-b0e1-224117c7ee1a.png)

We need something short, to show more profile name. `Profile exec:`, `Profile show:`.

Reply:
Maybe some unicode emoji icon type thing?

Reply:
Hmm, interesting idea. Nothing clear jumps to mind right now, though :/.

Reply:
![12](https://user-images.githubusercontent.com/62179190/190796549-7de0c6bc-6e50-42c3-9be6-dd1faf125c30.png)

![13](https://user-images.githubusercontent.com/62179190/190796630-9b2202b5-3c4f-4e8c-aee1-671239796280.png)

Reply:
Yeah, a checkmarked listcame to mind, not sure it's super clear though.

Did not think of the "shortcut/launcher" approach.

Oh, also just realized you meant to replace the *verb* by an icon, my train of thought was to replace "profile" itself, which was more annoying ;p.

TL;DR: I like the first one ;p.

Reply:
![14](https://user-images.githubusercontent.com/62179190/190807211-68d04b0a-4bb3-4ea8-8032-c8337451a316.png)

Reply:
I like ![image](https://user-images.githubusercontent.com/24273478/190806433-2cb70406-9ecd-49b0-b259-fef0a1fbeb0f.png) for the quickmenu ! Not so sure about the ones for exec :)
Remember we have most of https://www.nerdfonts.com/ - a bit tricky to get their codepoint though (some shift is needed, needs to look at https://github.com/koreader/koreader-fonts/pull/8#issuecomment-554145620 and some trials).

Reply:
Yeah, that could work. Plasma uses a similar approach for its "Open with" icon.

(I checked because I wanted to check what was used for the "Execute" entry on a shell script, but that's just the "terminal" angled bracket, which would possibly be more obscure for non-Linux/macOS users ;)).

Reply:
I mean, maybe don't elide "Profile" on submenus that don't directly come from a Profile-related thingy, though?

Reply:
> a bit tricky to get their codepoint though

FWIW, my usual approach is to get the name from the nerdfonts website, then look it up inside our own font w/ fontforge ;).

Reply:
I would keep the "Profile" word, it helps in the long list of dispatcher actions.

![16](https://user-images.githubusercontent.com/62179190/190815569-53b722d7-4cd8-43dc-92fa-0d75a5a4a453.png)

![17](https://user-images.githubusercontent.com/62179190/190815605-a2b9669f-9c40-4c18-8f2a-1cf0168e5494.png)

Reply:
I think we should add these icons next to the "Execute" and "Show as QuickMenu" so the user will recognize them immediately.